src/libs/fileDownload/index.android.js

```diff
@@ -0,0 +1,96 @@
+import {PermissionsAndroid} from 'react-native';
+import RNFetchBlob from 'rn-fetch-blob';
+import * as FileUtils from './FileUtils';
+
+/**
+ * Android permission check to store images
+ * @returns{Promise}
+ */
+function hasAndroidPermission() {
+    return new Promise((resolve, reject) => {
+        // read and write permission
+        const readPermission = PermissionsAndroid.PERMISSIONS.READ_EXTERNAL_STORAGE;
+        const writePermission = PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE;
```

Comment:
Please use them directly...
```suggestion
```

Reply:
Same, I didn't change as it existed beforehand.